package.json

```diff
@@ -85,7 +85,7 @@
     "typescript": "5.1.6"
   },
   "peerDependencies": {
-    "@typescript-eslint/parser": "^6 || ^7",
+    "@typescript-eslint/parser": ">=6",
```

Comment:
Not a maintainer, but as you suggested in the PR description, it's probably better to use `|| ^8` to avoid any weird behavior with a potential v9 that could break it "silently".

Besides, I've been testing it on v8 for a while, and it works well.

Reply:
(BTW I am not _at all_ trying to shame for it not being actively maintained, I know what I'm paying for it and am glad it's around)